config/config.yaml

```diff
@@ -48,23 +62,8 @@ branch-protection:
             - dco
             - pull-cert-manager-trust-verify
             - pull-cert-manager-trust-smoke
-
     jetstack:
-      # TODO: enforce dco check org-wide
-      # protect: true
-      # required_status_checks:
-      #   contexts:
-      #   - dco
-      repos:
-        cert-manager:
-          protect: true
-          required_status_checks:
-            contexts:
-            - dco
-            - pull-cert-manager-bazel
-            - pull-cert-manager-deps
-            - pull-cert-manager-chart
-            - pull-cert-manager-e2e-v1-23
+      repos: {}
```

Comment:
This whole `jetstack` entry can be removed now right?

Reply:
Done.